test/ParseableInterface/ModuleCache/prebuilt-module-cache-unusable.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %empty-directory(%t)
+// RUN: %empty-directory(%t/MCP)
+// RUN: %empty-directory(%t/prebuilt-cache/Lib.swiftmodule)
+
+// Prebuild a module for some imaginary CPU and put it in the prebuilt cache.
```

Comment:
This is pretty good, but I think in order to make sure the prebuilt cache is working at all, you should also add a positive test for the current CPU. That way if we change the path someday we won't get a false negative here.